project/src/main/webapp/voting_edu.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="utf-8">
+    <meta name="Voter Central Education Page"
+          content="This page is dedicated to educating voters on basic 
+              information about voting in the United States">
+    <link rel="stylesheet" href="style.css">
+    <script src="config.js"></script>
+    <script src="script.js"></script>
+    <title>Voter Central Education Page</title>
+  </head>
+  <body>
+    <h2>How to Vote</h2>
+    <iframe width="560" height="315" 
+        src="https://www.youtube.com/embed/
```

Comment:
Can you keep the iframe formatting consistent? e.g. here src is in a new line, in the next ones it's not.